web/pf4/src/components/ChartWithLegend.tsx

```diff
@@ -18,7 +18,8 @@ type Props = {
   fill?: boolean;
   stroke?: boolean;
   moreChartProps?: ChartProps;
-  overlay?: VCOverlay;
+  overlay?: Overlay;
+  onClick?: (datum: any) => void;
```

Comment:
I'm getting familiar with some classes, and some "any" types come to my attention.
Not for this PR, but perhaps that could be moved to a type in some iteration.
(But I need to read more the code to see the global context).

Reply:
+1